beep/protocol/maccor.py

```diff
@@ -1269,6 +1362,79 @@ def insert_driving_parametersv1(reg_params, waveform_directory=None):
     return file_path
 
 
+def insert_charging_parametersv1(reg_params, waveform_directory=None, voltage_limit_offset=0.01, max_c_rate=3.0):
+    """
+    Args:
+        reg_params (pandas.DataFrame): containing the following quantities
+            charge_type_1 (str): {'smooth', 'step'}
+            charge_current_param_1 (float): charge c-rate for first 20%
```

Comment:
@patrickherring-TRI Update comments (equally distributed windows over the charging range)